.npmrc

```diff
@@ -0,0 +1 @@
+registry='https://registry.npmjs.org/'
```

Comment:
Doesn't hurt, but I think the suggest was more for your local setup since it was reverting to the other repository

Reply:
Yeah, I prefer to specify it just because it can also happen to others.